test/e2e/playwright-multi-run/actor/main.js

```diff
@@ -0,0 +1,27 @@
+import { Actor } from 'apify';
+import { Dataset, PlaywrightCrawler } from '@crawlee/playwright';
+import { ApifyStorageLocal } from '@apify/storage-local';
+
+if (process.env.STORAGE_IMPLEMENTATION === 'LOCAL') {
+    await Actor.init({ storage: new ApifyStorageLocal() });
+} else {
+    await Actor.init();
+}
+
+const crawler = new PlaywrightCrawler({
+    maxRequestsPerCrawl: 10,
+    async requestHandler({ page, enqueueLinks, request }) {
+        const { url } = request;
+        const pageTitle = await page.title();
+        await Dataset.pushData({ url, pageTitle });
+        await enqueueLinks({
+            globs: ['**/3.0/examples/*'],
+        });
+    },
+});
+
+await crawler.run(['https://crawlee.dev/docs/3.0/examples/']);
+await crawler.run(['https://crawlee.dev/docs/3.0/examples/']);
+await crawler.run(['https://crawlee.dev/docs/3.0/examples/']);
```

Comment:
This is an interesting question. Is the point of re-running the ability to re-crawl or should the crawler remember the handled requests from previous run? It ties with the purging of the queue. I'm not sure this is the only valid use-case.

Reply:
I think that they should be using the `keepAlive` if they want the shared state. I don't think the use case is a long-running scrape, that makes no sense to do via some express handlers (which is kinda the thing this tries to solve).

But yeah, let's make this configurable. I would still keep the RQ purge enabled by default.